examples/ivr_outbound_dialer.py

```diff
@@ -0,0 +1,374 @@
+#Switchy IVR Outbound Dialer
+#
+#Author: Nenad Corbic <ncorbic@sangoma.com>
+#
+#This application is designed to demonstrate Switchy capabilities
+#and ease of use.  The application will connect to actively running
+#FreeSWITCH or Sangoma NSG application. It will then originate
+#a single call and play introductory IVR message. From then
+#on application will wait for user input via DTMF.
+#
+#IVR Menu
+# 911 - Play file contact system admin 
+# 811 - Play file hello
+# 111 - Cause a hangup
+# If user times out on DTMF a hello playback will be heard
+#
+#All user logic should be defined in IVRCallLogic Class.
+#
+#Variables
+#  self.<variables>       are global in nature.
+#  call.vars.['var_name'] should be used for per call info and state
+#
+#Switchy Documentation
+#  https://github.com/sangoma/switchy/blob/master/switchy/models.py
+#          class: Session Event Call Job
+#
+#  https://github.com/sangoma/switchy/blob/master/switchy/observe.py
+#          class: EventListener Client
+#
+#Sample Switchy Applications
+#  https://switchy.readthedocs.org/en/latest/apps.html
+#
+#License:
+#  BSD License
+#  http://opensource.org/licenses/bsd-license.php
+#
+#  Copyright (c) 2015, Sangoma Technologies Inc
+#  All rights reserved.
+#
+#  Redistribution and use in source and binary forms, with or without modification, 
+#  are permitted provided that the following conditions are met:
+#  1. Developer makes use of Sangoma NetBorder Gateway or Sangoma Session Border Controller
+#  2. Redistributions of source code must retain the above copyright notice, 
+#     this list of conditions and the following disclaimer.
+#  3. Redistributions in binary form must reproduce the above copyright notice, 
+#     this list of conditions and the following disclaimer in the documentation 
+#     and/or other materials provided with the distribution.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS 
+#  OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF 
+#  MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. 
+#  IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, 
+#  INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, 
+#  BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, 
+#  DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF 
+#  LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE 
+#  OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF 
+#  ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+
+import time
+import switchy
+import threading
+from switchy.marks import event_callback
+from switchy.apps.players import TonePlay
+from switchy import get_originator
+
+# Enable logging to stderr
+log = switchy.utils.log_to_stderr()
+# Debug levels: INFO in production, DEBUG in devel
+log.setLevel("INFO")
```

Comment:
You can pass the level to the initial call: `log = switchy.utils.log_to_stderr('INFO')`